src/modules/pages/components/ExpenditurePage/ExpenditurePage.css

```diff
@@ -20,5 +20,17 @@
 .sidebar {
   height: calc(100vh - navBarHeight);
   width: sidebarWidth;
+  overflow-y: auto;
   background-color: var(--grey-blue-3);
 }
+
+.sidebar::-webkit-scrollbar {
+  display: none;
+}
```

Comment:
We don't need to add vendor suffixes manually, as webpack runs a `postcss` step for use which does this automatically for the bundled code